### PR TITLE
Update Enable Inbound Traffic info

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by fumingshih — APPROVED** [review]

**Review by rddheath — APPROVED** [review]

File: modules/ROOT/pages/enable-inbound-traffic.adoc [hunk 1]
```diff
@@ -3,208 +3,131 @@ ifndef::env-site,env-github[]
...
+. In the *Basic Configuration* section, verify that *Secure Port* is set to `443` and that *Non Secure Port* is set to `80`.  The nonsecure port redirects to the secure port.
+. In the *Resource Allocation* section:
++
+The number of inbound load balancer replicas is determined by the number of controller nodes configured in Runtime Fabric. A minimum of three replicas is required for production environments, and these replicas run only on the controller VMs. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
++
+*Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified. Dedicated mode specifies that the entire node is available, so you cannot choose the amount of cores and memory. If Dedicated mode is disabled, then all nodes are already in sharing mode, which prevents them from being dedicated.
```
Comment (fumingshih) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]: Given that *Shared Mode* is in bold face, shouldn't "Dedicated mode" also be in bold face?

**Review by fumingshih — APPROVED** [review]
Please fix the bold-face for both "Shared Mode" and "Dedicated Mode" so they are consistent, other than that I am good.

File: modules/ROOT/pages/enable-inbound-traffic.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: Good catch - I've fixed this. Thank you!

File: modules/ROOT/pages/enable-inbound-traffic.adoc [hunk 2]
```diff
@@ -3,208 +3,131 @@ ifndef::env-site,env-github[]
...
+. In the *Basic Configuration* section, verify that *Secure Port* is set to `443` and that *Non Secure Port* is set to `80`.  The nonsecure port redirects to the secure port.
+. In the *Resource Allocation* section:
++
+The number of inbound load balancer replicas is determined by the number of controller nodes configured in Runtime Fabric. A minimum of three replicas is required for production environments, and these replicas run only on the controller VMs. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
++
+*Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified. *Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of cores and memory. If *Dedicated Mode* is disabled, then all nodes are already in sharing mode, which prevents them from being dedicated.
```
Comment (sundippatel) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]: Think we should split up shared and dedicated mode. It seems like there should be a break in between those options -- perhaps listed.

File: modules/ROOT/pages/enable-inbound-traffic.adoc [hunk 3]
```diff
@@ -3,208 +3,131 @@ ifndef::env-site,env-github[]
...
 
-== Assign Required Roles and Permissions
+.. Specify the minimum number of cores for each replica of the internal load balancer. When you select a Runtime Fabric from the Runtime Fabrics page, the sidebar that is displayed shows the total number of controller nodes deployed and total memory consumption for all deployed controller nodes. Use the sidebar information to help you specify values for the *Cores* and *Memory* fields for each replica.
```
Comment (sundippatel) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]: I think we wrote a comment to simplify this in the doc -- to:
 - explain how the internal load balancer runs on each controller node.
 - ask the user to specify the CPU cores and memory to allocate for each controller node.
 - remove the procedure to figure out how many controller nodes the cluster has.

**Review by sundippatel — DISMISSED** [review]

File: modules/ROOT/pages/enable-inbound-traffic.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: Thank you - I've updated this.

File: modules/ROOT/pages/enable-inbound-traffic.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: Thank you - I've updated this.

**Pull request merged**